### DISCUSSION TITLE
TypeError: stat: path should be string, bytes, os.PathLike or integer, not list

### DISCUSSION THREAD

**Discussion by pzzmyc** [d8ahazard/sd_dreambooth_extension#818] [Q&A]
anybody knows whats going on，Am I missing something？

Converting unet...
Exception compiling checkpoint: stat: path should be string, bytes, os.PathLike or integer, not list
Traceback (most recent call last):
  File "A:\AI\Stable_Diffushion_WebUI\stable-diffusion-webui\extensions\sd_dreambooth_extension\dreambooth\diff_to_sd.py", line 390, in compile_checkpoint
    if not os.path.exists(lora_path):
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\genericpath.py", line 19, in exists
    os.stat(path)
TypeError: stat: path should be string, bytes, os.PathLike or integer, not list

**Comment by AbyszOne** [comment]
Same. Saving a model after training.

**Comment by GigsTheCat** [comment]
By default now it already saves a model after training, so make sure it hasn't already saved a model for you.

**Reply by AbyszOne** [reply]
I stopped training and then generate. There it fails.

**Comment by David-337** [comment]
Same! unable to generate a ckpt with this error.

**EDIT: I was able to fix this** by going to the **db_config.json** in the dreambooth model's folder and changing
`"lora_model_name": [],`
to
`"lora_model_name": "",`

**Comment by gordingus** [comment]
This keeps happening to me as well. Whenever I change that .json and click Generate again, it gives the same error and the .json is reset to brackets.
